### ISSUE TITLE
PEP 8 compliance

### ISSUE THREAD

**Issue by davidlesieur** [datastore/datastore#12]
I have noticed that the code is not fully [PEP 8](http://www.python.org/dev/peps/pep-0008/) compliant. For instance, some method names use a mixed case form instead of lowercase with underscores. 

Are there any intentions to improve PEP 8 compliance? If so, how to handle backward compatibility? Thanks.

**Comment by jbenet** [comment]
@davidlesieur indeed, we should definitely be PEP 8 compliant. (this is an unfortunate holdover of hacking on js at the time of writing the bulk of this library :(, bad me)  Since you're reading through it already, feel free to flag things in this issue, or even submit patches.

Since we're still pre-1.0.0, we can break backward compatibility. of course, better not to. I think the focus right now should be on the best quality of library possible, and i think PEP 8 compliance trumps the small usage at the moment. Thoughts?
